settings.gradle

```diff
@@ -38,6 +40,14 @@ gradleEnterprise {
     }
 }
 
+dependencyResolutionManagement {
+    repositories {
+        mavenCentral()
+        maven{ url('https://repository.ow2.org/nexus/content/repositories/public/')}
```

Comment:
This block looks quite suspicious. Should never use `mavenLocal()` in any case, and the `ow2` repo seems redundant.

Reply:
Should we also remove `mavenLocal()` in `pluginManagement`?

Reply:
69867ef9509fa8bb74a0ba295db62c739a198f0b